backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5747190_ad_eventlog_eventName_as_list.sql

```diff
@@ -0,0 +1,538 @@
+-- 2025-02-20T12:52:34.345Z
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Reference_ID,AD_Ref_List_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541929,543860,TO_TIMESTAMP('2025-02-20 12:52:34.196000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'D','Y','AllEventsProcessedEvent',TO_TIMESTAMP('2025-02-20 12:52:34.196000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'AllEventsProcessedEvent','AllEventsProcessedEvent')
+;
+
+-- 2025-02-20T12:52:34.346Z
+INSERT INTO AD_Ref_List_Trl (AD_Language,AD_Ref_List_ID, Description,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Ref_List_ID, t.Description,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Ref_List t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Ref_List_ID=543860 AND NOT EXISTS (SELECT 1 FROM AD_Ref_List_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Ref_List_ID=t.AD_Ref_List_ID)
+;
+
+-- Reference: Material Event Names
+-- Value: StockEstimateDeletedEvent
+-- ValueName: StockEstimateDeletedEvent
+-- 2025-02-20T12:54:55.595Z
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Reference_ID,AD_Ref_List_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541929,543861,TO_TIMESTAMP('2025-02-20 12:54:55.413000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'D','Y','StockEstimateDeletedEvent',TO_TIMESTAMP('2025-02-20 12:54:55.413000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'StockEstimateDeletedEvent','StockEstimateDeletedEvent')
+;
+
+-- 2025-02-20T12:54:55.597Z
+INSERT INTO AD_Ref_List_Trl (AD_Language,AD_Ref_List_ID, Description,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Ref_List_ID, t.Description,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Ref_List t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Ref_List_ID=543861 AND NOT EXISTS (SELECT 1 FROM AD_Ref_List_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Ref_List_ID=t.AD_Ref_List_ID)
+;
+
+-- Name: Material Event Names
+-- 2025-02-20T12:57:09.494Z
+UPDATE AD_Reference SET IsOrderByValue='Y',Updated=TO_TIMESTAMP('2025-02-20 12:57:09.493000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Reference_ID=541929
+;
+
+-- Column: AD_EventLog.EventName
+-- 2025-02-20T12:58:56.689Z
+UPDATE AD_Column SET AD_Reference_ID=17, AD_Reference_Value_ID=541929,Updated=TO_TIMESTAMP('2025-02-20 12:58:56.689000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Column_ID=589675
```

Comment:
IMHO 
1. either you are keeping EventName as String
2. either you are making Event.eventName an enum too

personally I would go with option 1. if u don't have a very good reason of controlling what goes to event name

Reply:
reverted to using a string